README.md

```diff
@@ -1,45 +1,96 @@
 # starter-kit-storage-and-interfaces
 
-To boot strap all of this simply run:
+There exist two compose files at the root of the repo. Details on how to use them are provided below.
 
-```cmd
+## Starting the full stack with LS-AAI-mock
+
+To bootstrap the *full stack* of `storage-and-interfaces` services use
+the file `docker-compose.yml`. Note that this requires a running [`LS-AAI-mock`](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) service. To configure the LS-AAI-mock service follow the instructions below.
+
+Add the following line to `/etc/hosts` file:
+
+```
+127.0.0.1 aai-mock
+```
+
+First clone the [startet-kit-lsaai-mock](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) repo.
+
+Under its root folder, change the first two lines of the file `configuration/aai-mock/application.properties` to:
+
+```
+main.oidc.issuer.url=http://aai-mock:8080/oidc/
+web.baseURL=https://aai-mock:8080/oidc
+```
+
+and then add the `sda-auth` client by creating a file `configuration/aai-mock/clients/client1.yaml` with the following contents:
+
+```ini
+client-name: "auth"
+client-id: "XC56EL11xx"
+client-secret: "wHPVQaYXmdDHg"
```

Comment:
Although this secret has been used on the README of `sda-auth`, I'm still curious why use such a seemingly authentic password which is public on the internet.

Reply:
Well, this has been in the compose file of the neic sda-auth repo for ages and I copied it from there along with other stuff. In any case, I believe the key word here is "seemingly" :-)